content/blog/news/gsoc-2020-cloud-plugin.md

```diff
@@ -0,0 +1,155 @@
+    Deletion of the cloud group from the cloud provider.
+    
+    `Google Cloud Platform` implementation of this abstract class can be found here: [GoogleCloudGroup](https://github.com/AcademySoftwareFoundation/OpenCue/blob/d4a2eca93d271dacb13ca2aaba9b531642c1dc6b/pycue/opencue/cloud/gce_api.py#L22)
+ 
+### Plugin widget design
+
+The `CloudManager` is a plugin for CueCommander. There are **three** levels to implementing this entire widget:
+
+1. Creating the widget inside of the [plugins](https://github.com/AcademySoftwareFoundation/OpenCue/tree/526efee2fbf8c442a8e9d631e1806854463c4301/cuegui/cuegui/plugins) directory. 
+The module level variables to be defined for the plugin are mentioned in the docs of the [plugin registering script](https://github.com/AcademySoftwareFoundation/OpenCue/blob/526efee2fbf8c442a8e9d631e1806854463c4301/cuegui/cuegui/Plugins.py). 
+This helps registering the plugin, making it show up in the submenu and making it launch-able through the menu item.
+
+2. Implementing `AbstractDockWidget` for the [CloudManagerDockWidget](https://github.com/AcademySoftwareFoundation/OpenCue/blob/d4a2eca93d271dacb13ca2aaba9b531642c1dc6b/cuegui/cuegui/plugins/CloudManagerPlugin.py#L31) to create a dockable widget inside of the CueCommander interface. 
+
+3. The plugin itself then puts together two separate widgets:
+
+    * [CloudManagerTreeWidget](https://github.com/AcademySoftwareFoundation/OpenCue/blob/d4a2eca93d271dacb13ca2aaba9b531642c1dc6b/cuegui/cuegui/CloudManagerWidget.py#L63): 
+    This implements the [AbstractTreeWidget](https://github.com/AcademySoftwareFoundation/OpenCue/blob/526efee2fbf8c442a8e9d631e1806854463c4301/cuegui/cuegui/AbstractTreeWidget.py) class which provides very useful methods to handle the way data has 
+    to be displayed in the widget.
+    
+    * [CloudGroupCreateDialog](https://github.com/AcademySoftwareFoundation/OpenCue/blob/d4a2eca93d271dacb13ca2aaba9b531642c1dc6b/cuegui/cuegui/CloudGroupDialog.py#L28):
+    Which is used outside of the tree view and launched when the `Add Cloud Group` button is clicked on the plugin.
+    
+     
+The interface uses the main `CloudManager` api, mentioned above, as the entry point to then get all cloud groups
+associated with all the registered cloud providers.
+
+
+## Cloud plugin setup
+
+### Google compute engine setup
+
+{{% alert title="Note" color="info"%}}My deployment included setting up Cuebot on GCE{{% /alert %}}
+
+Listing the minimum requirements for the cloud plugin to work with GCE:
+
+1. GCP project with Compute Engine API and Cloud Logging API enabled.
+
+2. Setup [Application Default Credentials](https://cloud.google.com/docs/authentication/production) and make sure the google python api client (which is a requirement for this plugin)
+ has access to it.
+
+3. Use the `cloud_plugin_resources.yaml` under the `cuegui/config` directory to setup your GCP project name and
```

Comment:
Instead of "Use" I'd say "Modify" -- a bit more clear what you're asking them to do.

Also you can just link to `cloud_plugin_resources.yaml` in Github then drop the "under the cuegui/config directory".